freud/util/ManagedArray.h

```diff
@@ -90,7 +90,7 @@ template<typename T> class ManagedArray
     //! Return a constant pointer to the underlying data
     const T* data() const
     {
-        return m_data.data();
+        return m_data.data(); // TODO: not valid for std::vector<bool>
```

Comment:
I think we handled this in a different way, correct?
```suggestion
        return m_data.data();
```